package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Note managment tool",
   "main": "server.js",
   "scripts": {
-    "start": "babel-node server/server.js",
+    "start": "webpack-dev-server --port 3000",
+    "start:old": "babel-node server/server.js",
```

Comment:
let's delete `start:old` and `server/server/js`? It looks like it's doing the same thing as the new `npm start`.

Reply:
And then we can delete `server/logger.js` as well

Reply:
Yes. Will do that later when refactoring